tests/e2e/default/enketo/pregnancy.wdio-spec.js

```diff
@@ -0,0 +1,136 @@
+const moment = require('moment');
+const utils = require('../../../utils');
+const commonPage = require('../../../page-objects/default/common/common.wdio.page');
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+const userFactory = require('../../../factories/cht/users/users');
+const placeFactory = require('../../../factories/cht/contacts/place');
+const personFactory = require('../../../factories/cht/contacts/person');
+const contactPage = require('../../../page-objects/default/contacts/contacts.wdio.page');
+const tasksPage = require('../../../page-objects/default/tasks/tasks.wdio.page');
+const reportsPage = require('../../../page-objects/default/reports/reports.wdio.page');
+const analyticsPage = require('../../../page-objects/default/analytics/analytics.wdio.page');
+const genericForm = require('../../../page-objects/default/enketo/generic-form.wdio.page');
+const pregnancyForm = require('../../../page-objects/default/enketo/pregnancy.wdio.page');
+
+describe('Pregnancy registration', () => {
+  const places = placeFactory.generateHierarchy();
+  const healthCenter = places.find(place => place.type === 'health_center');
+  const offlineUser = userFactory.build({ place: healthCenter._id, roles: ['chw'] });
+  const pregnantWoman = personFactory.build({
+    date_of_birth: moment().subtract(25, 'years').format('YYYY-MM-DD'), 
```

Comment:
Thank you for this, I will definitely test all those different scenarios.
Is there a better way to do it? I was trying to not put a specific birthday for the pregnant woman because that will need to be updated in a few years, but I am not sure if there is a better way to do it.

Reply:
Hmm I don't know a better way to do it at the moment 
Maybe for now, try to not pick edges like 28/29 Feb or 30/31/01 of any month, also maybe not leave it to current date because my "today" can be one of those edges. You can play safe with a base date like 15 April 
 and then add years to that.